Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -223,4 +225,28 @@ function (FixerInterface $fixer) {
                 ->setInputCode(null)
         );
     }
+
+    /**
+     * @return string|null
+     */
+    protected function lintSource($source)
+    {
+        if (!getenv('LINT_TEST_CASES')) {
```

Comment:
why check the `env` (again) and not if the linter `!==null` (which is what method really needs to know rigth?)